templates/utility.macros.j2

```diff
@@ -1,3 +1,10 @@
+{% macro img() %}
+<img {% for k in kwargs %}
```

Comment:
I've never seen this technique for taking arbitrary arguments. Very interesting. 

Wonder if we should use `-%}` etc to remove white-space and line breaks from the output?